src/vm/eventpipesession.cpp

```diff
@@ -18,14 +18,16 @@ EventPipeSession::EventPipeSession(
     IpcStream *const pStream,
     EventPipeSessionType sessionType,
     EventPipeSerializationFormat format,
+    bool rundownSwitch,
     uint32_t circularBufferSizeInMB,
     const EventPipeProviderConfiguration *pProviders,
     uint32_t numProviders,
     bool rundownEnabled) : m_index(index),
                            m_pProviderList(new EventPipeSessionProviderList(pProviders, numProviders)),
                            m_rundownEnabled(rundownEnabled),
```

Comment:
I think you could use `rundownEnabled` instead (it was a plumbing done in previous PR).

Reply:
I agree the rundownEnabled arg doesn't appear to be used right now, I'd just ask to keep the terminology distinct. It appears the pattern we are following is:

rundownEnabled = is rundown active on this session right now?
rundownRequested = when the session disables, should rundown be enabled at that point?

Reply:
Yes, that's the current terminology and usage.

Reply:
Yes - Jose and I talked about this variable name and I'm hoping to change that...